Dockerfile

```diff
@@ -30,6 +30,11 @@ RUN curl -Lo vault.zip https://releases.hashicorp.com/vault/1.2.3/vault_1.2.3_li
 RUN unzip vault.zip
 RUN chmod +x vault
 
+#kubeval
+RUN curl -Lo tar xf kubeval-linux-amd64.tar.gz https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz
```

Comment:
please update to use a download link to the specific version, so it doesn't unexpectedly change version when we update the image:
https://github.com/instrumenta/kubeval/releases/download/0.14.0/kubeval-linux-amd64.tar.gz

Reply:
I realized you gave me edit access so i made the change :)